bootstrapvz/plugins/puppet/tasks.py

```diff
@@ -1,63 +1,124 @@
+import os
 from bootstrapvz.base import Task
 from bootstrapvz.common import phases
-from bootstrapvz.common.tools import sed_i
-import os
+from bootstrapvz.common.exceptions import TaskError
+from bootstrapvz.common.releases import jessie, wheezy
+from bootstrapvz.common.tools import sed_i, log_check_call
+
+# This is an updated puppet module version, introducing the puppet 4 agent
+# and also enabling you to install puppet modules on the image ready.
+# For now it ASSUMES the following:
+# - Debian Wheezy Jessie install
+# - You want the puppet 4 agent (1.9.+) installed from apt.puppetlabs.com
+#
+# Modules installed with this plugin are "--force install"
 
 
 class CheckAssetsPath(Task):
-    description = 'Checking whether the assets path exist'
+    description = 'PUPPET: Checking whether the assets path exist'
```

Comment:
Prefixing task descriptions with the module name is non-idiomatic. Please remove or paraphrase to something like "Checking whether the puppet assets path exists"